engines/wintermute/base/base_persistence_manager.h

```diff
@@ -37,6 +37,10 @@
 #include "common/str.h"
 #include "common/system.h"
 #include "common/rect.h"
+#ifdef ENABLE_WME3D
```

Comment:
You don't need the headers, only a forward declaration. (the persistence-manager is included _everywhere_ so it's worthwhile to keep its dependencies as small as possible)